editor/editor_sectioned_inspector.cpp

```diff
@@ -266,7 +269,8 @@ void SectionedInspector::update_category_list() {
 			if (!section_map.has(metasection)) {
 				TreeItem *ms = sections->create_item(parent);
 				section_map[metasection] = ms;
-				ms->set_text(0, sectionarr[i].capitalize());
+				ms->set_text(0, EditorStringProcessor::get_singleton()->process_name(sectionarr[i]));
+				ms->set_tooltip(0, sectionarr[i]);
```

Comment:
I would set a tooltip text to the translated name (or translated and raw names on the two lines) when `translate_properties` is `false`. This way, translation will be always available, even if you prefer raw names in the interface.